services/workflow-manager/app/api/workflow.py

```diff
@@ -21,19 +36,115 @@
 # 404 = Not found
 
 
+SEQUENCE_MANAGER_URI = "host.docker.internal:8003"
+EXAM_MANAGER_URI = "host.docker.internal:8004"
+
+
 class RecoJob(BaseModel):
```

Comment:
Still required?

Reply:
Only as an intermediate step. Removed it now and refactored one step further.